.dockerignore

```diff
@@ -0,0 +1,26 @@
+# we try to aoid adding files to the docker images that change often
+# or that are not needed for running the docker image
+# tis greatly reduces the amount of times we need to rerun `npm install` when building image locally
+# https://codefresh.io/blog/not-ignore-dockerignore/
+# https://docs.docker.com/engine/reference/builder/#dockerignore-file
+
+# consider them hidden
+.*
```

Comment:
Ok prefect that's valid for me then 👍  I'll notify @Meeeeow he is working in  a ui branch.